migration/sql-files/006-external-provider.sql

```diff
@@ -0,0 +1,22 @@
+
+-- token storage
+
+CREATE TABLE external_providers (
+    id uuid primary key DEFAULT uuid_generate_v4() NOT NULL,
+    created_at timestamp with time zone,
+    updated_at timestamp with time zone,
+    deleted_at timestamp with time zone,
+    type text,
+    url text
+);
+
+CREATE TABLE external_provider_tokens (
+    id uuid primary key DEFAULT uuid_generate_v4() NOT NULL,
+    created_at timestamp with time zone,
+    updated_at timestamp with time zone,
+    deleted_at timestamp with time zone,
+    external_provider_id uuid  REFERENCES external_providers (id),
+    identity_id uuid REFERENCES identities(id),
+    token text,
+    scope text
```

Comment:
Let's not include scope for now. We will add it later when we support it.

Reply:
I was planning to save the default scope(s) for github & OSO now. So that, adding support later on will be more seamless to ensure existing data is already in line with the scope specification.